### DISCUSSION TITLE
Problems with core 2022.11.1

### DISCUSSION THREAD

**Discussion by bengtner** [wlcrs/huawei_solar#172] [General]
Has anyone managed to upgrade to core 2022.11 1 with this integration on? I made an upgrade this afternoon, and after that this integration fails. 

Logger: homeassistant.setup
Source: setup.py:184
First occurred: 22:41:36 (1 occurrences)
Last logged: 22:41:36

Setup failed for custom integration huawei_solar: Requirements for huawei_solar not found: ['huawei-solar==2.1.2'].

**Comment by tdr-its42** [comment]
No problems here, updated from last version of 2022.10 to 2022.11.1

**Reply by bengtner** [reply]
Hm, strange. I looked a bit deeper in my logs and found this as well:
```
Logger: homeassistant.util.package
Source: util/package.py:98
First occurred: November 4, 2022 at 22:41:21 (3 occurrences)
Last logged: November 4, 2022 at 22:41:36

Unable to install package huawei-solar==2.1.2: ERROR: Cannot install huawei-solar==2.1.2 because these package versions have conflicting dependencies. ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts [notice] A new release of pip available: 22.2.1 -> 22.3 [notice] To update, run: pip install --upgrade pip
```
Seems to be related to HASSIO and I noticed I was a bit behind on 9.0. Latest version 9.3. So I made an update of HASSIO, but the problem with setup of a huewer_solar is still there. Downgraded to 2022.10.5 and the problem is gone. I have no clue why this happens....

**Comment by wlcrs** [comment]
Duplicate of #163

**Reply by bengtner** [reply]
Thanks, forgot this is a custom component..... 
The good thing this forced me to learn how to switch back and forth between core versions. Not hard at all!
